include/TrustWalletCore/TWHDWallet.h

```diff
@@ -49,6 +52,10 @@ TWData *_Nonnull TWHDWalletSeed(struct TWHDWallet *_Nonnull wallet);
 TW_EXPORT_PROPERTY
 TWString *_Nonnull TWHDWalletMnemonic(struct TWHDWallet *_Nonnull wallet);
 
+// Wallet entropy
+TW_EXPORT_PROPERTY
+TWData *_Nonnull TWHDWalletEntropy(struct TWHDWallet *_Nonnull wallet);
```

Comment:
let's change `TWHDWalletCreateWithData` to `TWHDWalletCreateWithEntropy` as well

Reply:
Changed, this is breaking change too.